app/controllers/admin/orders_controller.rb

```diff
@@ -20,7 +20,7 @@ def index
           @orders = @orders.page(params[:page]).per(@query_params[:per_page])
         end
         format.csv do
-          @order_items = find_order_items(@orders.map(&:id))
+          @order_items = Orders::OrderItems.find_order_items(@orders.map(&:id), current_user)
```

Comment:
I would use `@orders.pluck(:id)` if possible instead of map, looks like it should work.